Description: Use system libs in tests
 Use the libs which are in $LOAD_PATH instead of using the relative paths.
Author: Jongmin Kim <jmkim@pukyong.ac.kr>
Forwarded: not-needed
Last-Update: 2020-09-27
--- a/test/redis/synchrony_driver.rb
+++ b/test/redis/synchrony_driver.rb
@@ -3,8 +3,8 @@
 require "em-synchrony"
 require "em-synchrony/connection_pool"
 
-require_relative "../lib/redis"
-require_relative "../lib/redis/connection/synchrony"
+require_relative "redis"
+require_relative "redis/connection/synchrony"
 
 require "helper"
 
